dotcom-rendering/src/components/Pill.tsx

```diff
@@ -0,0 +1,79 @@
+import { css } from '@emotion/react';
+import { space, textSansBold12 } from '@guardian/source/foundations';
+import type { ReactElement } from 'react';
+import { palette } from '../palette';
+
+type IconSide = 'left' | 'right';
+
+interface Props {
+	content: string | ReactElement;
+	prefix?: string;
+	icon?: ReactElement;
+	iconSide?: IconSide;
+}
+
+const pillStyles = css`
+	display: inline-flex;
+	align-items: center;
+	gap: ${space[1]}px;
+	padding: 0 10px;
+	border-radius: ${space[3]}px;
+	${textSansBold12};
+	color: ${palette('--pill-text')};
+	background-color: ${palette('--pill-background')};
+`;
+
+const pillContentStyles = css`
+	padding: ${space[1]}px 0;
+`;
+
+const pillPrefixStyles = css`
+	margin-right: 2px;
+	padding-right: 6px;
+	border-right: 1px solid ${palette('--pill-divider')};
+`;
+
+export const VideoIcon = () => (
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
+		height="10"
+		viewBox="0 0 8 10"
+		focusable={false}
+		aria-hidden={true}
+	>
+		<path
+			fill="currentColor"
+			fill-rule="evenodd"
+			clip-rule="evenodd"
+			d="M7.8 5.2748v-.5166L.2773-.002 0 .3302v9.3726l.2773.2952L7.8 5.2748Z"
+		/>
+	</svg>
+);
```

Comment:
Annoyingly the play icon in Source is subtly different to this one, but I'll ask Alex if we can use that instead as it would be more consistent. (Which is the whole point of having a design system and icon library!) The Source icons include a lot of extra whitespace too, but that can be worked around.

Reply:
This is now using Source icons 🎉  As discussed earlier, it turns out that the play icon _has_ been updated in the design system in Figma, but the Source component library hasn't been updated to reflect these changes. As a temporary measure, I've included a local copy of `SvgMediaControlsPlay` with the new design which can be removed once a new version of Source is available.